README.md

```diff
@@ -2,38 +2,37 @@
 
 This service allows ward staff to schedule a visit for a patient. Allowing face to face visits for visitors who are unable to visit in person.
 
+It includes an administration panel which provides self-service functionality for adding new Trusts, Hospitals, and Wards.
```

Comment:
Should we also update our glossary with these terms and pages? 🤔

Reply:
Yap!

Reply:
I've removed the pages, as I'm not convinced there's enough value documenting those anymore (we initially added it to keep the routes consistent)